### PR TITLE
Rootpage

### PR DESCRIPTION
**Pull request by maxbischof**
Implements the root page with welcome section and list of all datasets. https://github.com/maxbischof/data-hub-nord/issues/2

### CODE REVIEW

File: src/components/DatasetImage.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+export default function DatasetImage({ path }) {
+  return <Image src={path}/>
+}
+
+const Image = styled.img`
+  border-radius: 5px;
+  border: 1px solid var(--grey);
+  width: 100%;
+  margin: 0 0 0 0;
```
Comment (carolinwagner) [inline src/components/DatasetImage.js:13]: "margin: 0" is a little shorter ;-)

File: src/components/WelcomeSection.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  align-items: center;
+  flex-shrink: 0;
+`
+
+const Headline = styled.h1`
+  color: white;
+  font-size: 35px;
+  font-weight: 100;
+  text-align: center;
+  padding: 0 10px 0 10px;
```
Comment (carolinwagner) [inline src/components/WelcomeSection.js:40]: "padding: 0 10px" will also do the trick

File: src/components/WelcomeSection.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  margin: 120px 0 10px 0;
+`
+
+const SubHeadline = styled.h2`
+  color: white;
+  font-size: 20px;
+  text-align: center;
+  font-weight: 400;
+  margin: 0 0 50px 0;
+  padding: 0 10px 0 10px;
```
Comment (carolinwagner) [inline src/components/WelcomeSection.js:50]: "padding: 0 10px"

File: src/components/pages/RootPage.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react'
+import WelcomeSection from '../WelcomeSection'
+import DatasetList from '../DatasetList'
+import PropTypes from 'prop-types'
+
+export default function RootPage({ datasets }) {
+  const [datasetListRef, setDatasetListRef] = useState()
+
+  const scrollToRef = (ref) => {
+    console.log("scroll")
```
Comment (carolinwagner) [inline src/components/pages/RootPage.js:10]: console.logs should be removed before final deployment

File: src/components/ui/WelcomeButton.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+export default function Button({ color, children, onClick, reference }) {
+  return <StyledButton onClick={() => onClick(reference)} color={color}>{children}</StyledButton>
+}
+
+const StyledButton = styled.button`
+  font-size: 15px;
+  font-weight: 700;
+  text-decoration: none;
+  color: #fff;
```
Comment (carolinwagner) [inline src/components/ui/WelcomeButton.js:12]: Maybe you could also use the CSS variables as you already did in your GlobalStyles.js for reasons of consistency

File: cypress/plugins/index.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+/**
+ * @type {Cypress.PluginConfig}
+ */
```
Comment (carolinwagner) [inline cypress/plugins/index.js:17]: Comments should be removed if they aren't useful to understand your code

File: cypress/support/commands.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+// -- This is a child command --
+// Cypress.Commands.add("drag", { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add("dismiss", { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
```
Comment (carolinwagner) [inline cypress/support/commands.js:25]: I guess you can also remove these comments

File: cypress/support/index.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
```
Comment (carolinwagner) [inline cypress/support/index.js:16]: could be also deleted

File: src/components/Footer.js [hunk 9]
```diff
@@ -36,6 +37,5 @@ const StyledFooter = styled.footer`
     align-items: center;
     flex-direction: column;
     padding: 0 15px 0 15px;
```
Comment (carolinwagner) [inline src/components/Footer.js:39]: "padding: 0 15px" is a little shorter

File: src/components/WelcomeSection.js [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+import PropTypes from 'prop-types'
+
+export default function WelcomeSection({ onClickButton, scrollTo }) {
+  return (
+    <StyledWelcomeSection>
+      <Headline>Entdecke Open Data in Norddeutschland</Headline>
+      <SubHeadline>
+        Visualisierung und Zugang zu den wichtigsten Daten
+      </SubHeadline>
+      <WelcomeButton color="#e63946" onClick={onClickButton} reference={scrollTo}>Daten anzeigen</WelcomeButton>
```
Comment (carolinwagner) [inline src/components/WelcomeSection.js:14]: Maybe you can use CSS variables for reasons of consistency and more readability

**Review by carolinwagner — COMMENTED** [review]
Looks very good to me - I just added some minor comments :-)

File: src/components/pages/RootPage.js (hunk 4 above)
Comment (maxbischof) [reply src/components/pages/RootPage.js:10]: Missed that one. Thanks!

File: src/components/WelcomeSection.js (hunk 10 above)
Comment (maxbischof) [reply src/components/WelcomeSection.js:14]: good idea! Thanks.

**Pull request merged**